compiler/rustc_mir_build/src/build/coverageinfo/mcdc.rs

```diff
@@ -217,7 +217,7 @@ impl MCDCInfoBuilder {
                 }
                 _ => {
                     // Do not generate mcdc mappings and statements for decisions with too many conditions.
-                    let rebase_idx = self.branch_spans.len() - decision.conditions_num + 1;
+                    let rebase_idx = self.branch_spans.len() + 1 - decision.conditions_num;
```

Comment:
A comment about the ordering here seems warranted.

Reply:
After considering what to write for the comment, I noticed that in context this would be better off subtracting 1 from the RHS instead.